frontend/src/index.css

```diff
@@ -310,6 +311,19 @@ Other button classes are defined further down together with other classes for th
   color: hsl(185deg 92% 16%);
 }
 
+.login-img {
+  margin-bottom: 1rem;
+  width: 5.5rem;
+  height: auto;
```

Comment:
The default value for height is auto so this maybe doesn't make a difference?

Reply:
Yes, you're right!